packages/grafana-data/src/types/pluginExtensions.ts

```diff
@@ -117,6 +117,8 @@ export type PluginExtensionEventHelpers<Context extends object = object> = {
 export enum PluginExtensionPoints {
   AlertInstanceAction = 'grafana/alerting/instance/action',
   AlertingHomePage = 'grafana/alerting/home',
+  AlertingRuleAction = 'grafana/alerting/alertingrule/action',
+  RecordingRuleAction = 'grafana/alerting/recordingrule/action',
```

Comment:
I would probably suggest renaming this to something like `AlertingRecordingRuleAction`, so it's prefixed by `Alerting` and is easier to discover for plugin devs who are relying on the `PluginExtensionPoints` enum using static typing.

Reply:
Yeah, I just tried to avoid using `AlertingAlertingRuleAction` 😅